catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/DurableWebDavFileConsumer.java

```diff
@@ -130,7 +130,11 @@ public void onDirectoryDelete(DavEntry entry) {
 
     @Override
     public void onFileDelete(DavEntry entry) {
-      processExchange(getExchange(entry, StandardWatchEventKinds.ENTRY_DELETE));
+      processExchange(
+          getExchange(
+              new File(entry.getLocation()),
```

Comment:
❗️ This change doesn't make sense to me. `getLocation()` will return a remote URI, which will never be resolvable as a `File`, plus it's been deleted anyways. I think you are just trying to bypass [this line](https://github.com/peterhuffer/ddf/blob/3b434dc20a5de287a5a4b7461558b139491debc2/catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/DurableWebDavFileConsumer.java#L85) but I don't think this is the right way to do that. There's already a null check [here](https://github.com/peterhuffer/ddf/blob/3b434dc20a5de287a5a4b7461558b139491debc2/catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/DurableWebDavFileConsumer.java#L85), so I think you are better off just passing null.

Reply:
Yes, I was trying to pass that before, but I was getting an NPE down in Camel. I will see if I can find another way to get around that NPE.

Reply:
I will also take a look on my end.